### PR TITLE
RHTE Keynote deployment

### PR DESCRIPTION
**Pull request by pranavgaikwad**

### CODE REVIEW

File: demos/2019_RHTE/keynote/create_rhte_env.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    echo "Please ensure that 'AGNOSTICD_HOME' is set before running."
+    exit
+fi
+
+OUR_DIR=`pwd`
+pushd .
+cd ${AGNOSTICD_HOME}
+
+echo "Creating OCP3 env..."
+ansible-playbook ${AGNOSTICD_HOME}/ansible/main.yml -e @${OUR_DIR}/3.x/my_vars.yml -e @${OUR_DIR}/3.x/ocp3_vars.yml -e @${OUR_DIR}/secret.yml
```
Comment (jwmatthews) [inline demos/2019_RHTE/keynote/create_rhte_env.sh]: Assuming these invocations are serial currently?
We'd want to invoke the creation of 3.x and 4.x envs in parallel.

Comment (pranavgaikwad) [reply demos/2019_RHTE/keynote/create_rhte_env.sh]: yes that makes sense

Comment (pranavgaikwad) [reply demos/2019_RHTE/keynote/create_rhte_env.sh]: @jwmatthews good to merge with https://github.com/fusor/mig-agnosticd/pull/55/commits/7de8486e24f298695cd7c579c0e758919eab9e1a

**Comment by jwmatthews** [comment]
@pranavgaikwad FYI, brought in Todd's PR for lab docs
https://github.com/fusor/mig-agnosticd/pull/58

will need to rebase this

**Pull request merged**